admin/RunExport

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
The name of the script is my fault (from 17 years ago) -- let's not copy that bad name and go with something more sensical like create-dumps.sh or something...